CMakeLists.txt

```
@@ -3,8 +3,21 @@ project(benchmark_gazebo)
 enable_testing()
 
 include(FindBoost)
+find_package(PkgConfig REQUIRED)
+pkg_check_modules(zstd REQUIRED IMPORTED_TARGET libzstd)
+pkg_check_modules(lz4 REQUIRED IMPORTED_TARGET liblz4)
+find_package(Protobuf CONFIG)
```

```suggestion
find_package(protobuf CONFIG)
```
this works if I change the capitalization

---

oh, that's nice, it works on Ubuntu 22 too, I think the issue is with the version of protobuf, the official [documentation](https://cmake.org/cmake/help/latest/module/FindProtobuf.html) uses the capitalized version in CMake. Can I know which protobuf version you are using?

---

I'm using protobuf from homebrew-core on macOS, which is currently version 26.1 on my machine, and will be 27.0 as soon as I upgrade